scheduler/src/cook/mesos/monitor.clj

```diff
@@ -53,32 +47,32 @@
                        stats (-> job-ent
                                  util/job-ent->resources
                                  (select-keys [:cpus :mem])
-                                 (assoc :count 1))]
+                                 (assoc :jobs 1))]
                    {user stats})))
          (reduce (partial merge-with (partial merge-with +)) {}))))
 
-(defn add-aggregated-stats
+(defn- add-aggregated-stats
   "Given a map from users to their stats, associcate a special user
    \"all\" for the sum of all users stats."
   [db stats]
   (if (seq stats)
     (->> (vals stats)
          (apply merge-with +)
          (assoc stats "all"))
-    {"all" (zipmap (conj (util/get-all-resource-types db) :count)
-                   (repeat 0.0))}))
+    {"all" {:cpus 0, :mem 0, :jobs 0}}))
```

Comment:
Why not use `util/get-all-resource-types` to get all the resources? Isn't this list missing `:gpus`?

Reply:
The `:gpus` were already not getting tallied up due to this `(select-keys [:cpus :mem])`:

https://github.com/twosigma/Cook/pull/710/files#diff-09dd7646946b40f7cce49e291e1897cfL55

We could add `:gpus`, but I would vote to make that a separate enhancement PR rather than include the change here.

Reply:
sgtm